orga/translations/en.json

```diff
@@ -31,6 +32,10 @@
     }
   },
   "pages": {
+    "certifications": {
+      "description": "Jiberrish",
```

Comment:
La trad est dispo : You can export the certification results for all classes in a csv file.